README.md

```diff
@@ -2,17 +2,60 @@
 
 This is a sample [Marathon](https://github.com/mesosphere/marathon) app for encrypting your [Marathon-lb](https://github.com/mesosphere/marathon-lb) HAProxy endpoints using [Let's Encrypt](https://letsencrypt.org/). With this, you can automatically generate and renew valid SSL certs with Marathon-lb.
 
+The work done to support wildcard certificates and the DNS providers, was influenced heavily by the work done by Sebastian Woehrl ([https://github.com/MaibornWolff/letsencrypt-marathon-lb](https://github.com/MaibornWolff/letsencrypt-marathon-lb)).
```

Comment:
would be nice to know the differences from the project it was heavily inspired by in words like he does in his project.